handbook/sales/README.md

```diff
@@ -143,6 +143,40 @@ To schedule an [ad hoc meeting](https://www.vocabulary.com/dictionary/ad%20hoc)
  - Ensure that the product category is defined ("Endpoint ops", "Device management", or "Vulnerability management") in the description of the issue.
 
 
+### Conduct a technical evaluation (aka proof of value, or POV)
```

Comment:
- Let's use the same name that we use in Salesforce to keep it consistent across the tools (e.g. "Conduct a POV")